### DISCUSSION TITLE
missing config.json file

### DISCUSSION THREAD

**Discussion by heapxor** [dani-garcia/vaultwarden#2701] [General]
hello,
i installed vaultwarden, created account .. logged in etc... but i cant see any config.json file ... as described here https://github.com/dani-garcia/vaultwarden/wiki/Configuration-overview

also i didnt export any env variable as stated here > DOMAIN=https://bitwarden.example.com

any idea? thanks!

**Comment by Masgalor** [comment]
Hi.
The link you posted says
>you can perform configuration either via environment variables or an [admin page](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page) (which writes settings to a config.json file under your data directory).

So, did you enable the admin page and create a file that way, did you look in the right place?
Maybe the better question is, why are you looking for this file, there is nothing you can do with it.

>also i didnt export any env variable as stated here > DOMAIN=https://bitwarden.example.com/

Im not sure what you are saying here.
You need to set some settings with env variables, e. g. to enable the admin page.
But also to change variables that are marked read-only in the admin page.
